src/main/java/org/oasis_eu/portal/core/dao/InstanceACLStore.java

```diff
@@ -34,4 +34,36 @@
      * @param userIds
      */
     void saveACL(String instanceId, List<User> userIds);
+
+    /**
+     * Create ACL to allow an user to use an instance.
+     *
+     * @param instanceId
```

Comment:
Don't put such annotations if you have nothing to say about the param (and even if there had some before)